SU-SWS/stanford_capx
# DEVOPS-269: Update guzzle

# READY FOR REVIEW

# Summary
- Fix fatal error

# Needed By (Date)
- ASAP? Not sure how pervasive the error is.

# Criticality
- How critical is this PR on a 1-10 scale? 4/10?

# Steps to Test

1. Check out the [DEVOPS-269 branch](https://github.com/SU-SWS/acsf-cardinald7/tree/DEVOPS-269) on local and clone `jumpstartwoods` to local, and run `drush cron`, **OR**
2. run `drush cron` on ACSF dev on the `jumpstartwoods` site: `drush @acsf.dev.cardinald7.jumpstartwoods cron`
3. Get no fatal error (there are other warnings)

# Affected Projects or Products
- D7

# Associated Issues and/or People
- JIRA ticket: `DEVOPS-269`

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
